lib/prompts.ts

```
@@ -0,0 +1,139 @@
+    : "👋 Welcome to Study Chat! I'm here to help you learn.\n\n" +
+      "What would you like to study today?",
+});
+
+/**
+ * System prompt for the general chat assistant.
+ * This is used when the user is chatting outside of any specific module.
+ *
+ * @param searchContext - Optional search results to include in the prompt
+ * @returns The system prompt string
+ */
+export const createGeneralSystemPrompt = (searchContext: string = "") => `
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)